shell/pom.xml

```diff
@@ -98,5 +102,11 @@
       <type>test-jar</type>
       <scope>test</scope>
     </dependency>
+    <dependency>
```

Comment:
However I'm not sure if  this is desirable

Reply:
I believe generally we want to separate the client and server so one does not depend on the other. If some logic is shared, I think what we do is to extract that into somewhere like `alluxio-core-common` so both can use it.

Reply:
If we extract the raft related logic to `alluxio-core-common` which means ratis dependencies will involved into alluxio client.